models/transactions/transaction.go

```diff
@@ -61,15 +61,15 @@ func NewTransaction(clientID, partnerID string, amount int) (*Transaction, error
 		return nil, ErrMissingPartnerID
 	}
 
-	if amount < 0 {
```

Comment:
I think 0 is a valid amount as it could mean a free trip for whatever reason

Reply:
0 is a not valid transaction in most payments APIs